includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -595,7 +595,7 @@ public function is_available_for_current_currency() {
 		$supported_currencies = $this->account->get_account_customer_supported_currencies();
 		$current_currency     = strtolower( get_woocommerce_currency() );
 
-		if ( count( $supported_currencies ) === 0 ) {
+		if ( ! is_array( $supported_currencies ) || array() === $supported_currencies ) {
```

Comment:
Based on the PHPCS results, it looks like this should be `[]` instead of `array()`.
```suggestion
		if ( ! is_array( $supported_currencies ) || [] === $supported_currencies ) {
```